connect/runtime/src/main/java/org/apache/kafka/connect/runtime/HerderConnectorContext.java

```diff
@@ -63,6 +64,11 @@ public void raiseError(Exception e) {
         herder.onFailure(connectorName, e);
     }
 
+    @Override
+    public PluginMetrics pluginMetrics() {
+        return null;
+    }
+
```

Comment:
If we don't want this, we could define `pluginMetrics()` in `SourceConnectorContext` and `SinkConnectorContext`

Reply:
I actually moved the `PluginMetricsImpl` field in this class.